x-pack/plugins/enterprise_search/public/applications/shared/result/result.scss

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+.resultField:nth-child(odd) {
```

Comment:
question: This is a looooot of CSS for us. Is there no way to do this with 'plain' components?